### PR TITLE
Release/1.3.0

### PR DESCRIPTION
**Pull request by dragos-dobre**

### CODE REVIEW

File: _include/um/User_login/forgotten_password.rst [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+  #. Click on ``Confirm password`` link inside the email recieved with the subject ``IMIS Password Request`` then re-enter the data of the steps 1 & 2 (:ref:`New password confirmation image <confirm_pass>`).
+
+     .. _confirm_pass:
+     .. figure:: /img/user_manual/confirm_pass.png
+        :align: center
+
+        `Image - New password confirmation`
+
+  #. the new credentials can be use in the `Login <#login>`__ page.
```
Comment (kmadej) [inline _include/um/User_login/forgotten_password.rst]: The new credentials can be used...

File: _include/um/User_login/password_change.rst [hunk 2]
```diff
@@ -0,0 +1,52 @@
+Password change
+^^^^^^^^^^^^^^^
+
+  Any user can change his/her password by adjustment of his/her profile.
+
+Navigation
+""""""""""
+
+  Functionality for changing of a password can be in the menu ``My Profile``, sub menu ``Change Password``
```
Comment (kmadej) [inline _include/um/User_login/password_change.rst]: something wrong in this sentence. I see two possibility for update:
1. Functionality for changing of a password can be done in the menu...
2. Functionality of changing of a password is in the menu...

File: _include/um/User_login/password_change.rst [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+      Enter a new password of the current user. The password should have at least 8 alphanumeric characters with at least one digit.
+
+    * ``Confirm Password``
+
+      Repeat the new password of the current user.
+
+ #. **Saving**
+
+    Once all mandatory data is entered, clicking on the ``Save`` button will save the record. The user will be re-directed back to the `Home Page <#image-2.2-home-page>`__. A message confirming that the new password has been saved will appear at the bottom.
```
Comment (kmadej) [inline _include/um/User_login/password_change.rst]: all mandatory data are entered.

Edit: I googled it and it seems that both options data are/is are corrected, however in my opinion are sounds better here

File: _include/um/claims/batch_claims.rst [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+
+    The batch processing panel allows a user to process batches based on the following criteria:
+
+    * ``Region``
+
+      Select the ``Region``; from the list of regions by clicking on the arrow on the right of the selector to select a region. *Note: The list will only be filled with the regions assigned to the current logged in user and the option National.*
+
+    * ``District``
+
+      Select the ``district``; from the list of districts by clicking on the arrow on the right of the selector to select a district. *Note: The list will only be filled with the districts belonging to the selected region and assigned to the current logged in user. If this is only one then the district will be automatically selected. If no district is selected then the processing is done only for insurance product defined for the selected region.*
```
Comment (kmadej) [inline _include/um/claims/batch_claims.rst]: if this is the only one

File: _include/um/claims/hf_claims.rst [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+
+      Enter the insurance number of the patient. When done entering this field, the corresponding name of the patient will be filled on the name of the patient (the text box which is read only field and is on the right side of the Insurance Number text field). Mandatory.
+
+    * ``Claim No.``
+
+      Enter the identification of the claim. Mandatory, up to 8 characters. It should be unique within the claiming health facility.
+
+    * ``Main Dg.``
+
+      Select the code of the main diagnosisby typing text, all diagnoses containing the typed text will appear and be selectable underneath the box. Mandatory.
```
Comment (kmadej) [inline _include/um/claims/hf_claims.rst]: diagnosis by

File: _include/um/claims/review_claims.rst [hunk 6]
```diff
@@ -0,0 +1,594 @@
...
+
+      Once the criteria have been entered, use the search button to filter the records, the results will appear in the Result Panel.
+
+ #. **Claim Selection Update Panel**
+
+    This panel is basically for functionality of updating multiple claims which are currently loaded in the Result Panel at once basing on the claim filter criteria available on this panel. The update on the claims is basically changing **Feedback Status** and **Review Status** of a claim from **Idle** to (**Not**) **Selected for Feedback** or (**Not**) **Selected for Review** respectively. The filters in this panel work on the claims which are currently loaded on the result panel. The combination of filters is either ``Select`` alone or ``Select`` and either ``Random`` or ``Value`` or ``Variance`` or combination of ``Value`` and ``Variance``.
+
+    * ``select``
+
+      Is a selection dropdown box to select between **Review Select** and **Feedback Select** to filter only claims whose review status is **Idle** or feedback status is **Idle** respectively from among claims currently in the Result Panel.
```
Comment (kmadej) [inline _include/um/claims/review_claims.rst]: not sure if sentence should start with 'Is'

File: _include/um/claims/review_claims.rst [hunk 7]
```diff
@@ -0,0 +1,594 @@
...
+ #. **Button Panel**
+
+    With exception of the Cancel button, which re-directs to the `Claims Overview Page <#claims-overview-page>`__, the button panel is used in conjunction with the current selected record (highlighted with blue). The user should first select a record by clicking on any position of the record.
+
+    * ``review``
+
+      Clicking on this button re-directs a user to the `Claim Review Page <#claim-review-page>`__, where a claim with review status **Selected for Review** can be reviewed and its current review status changed to **Reviewed.** If the claim is not in the status **Selected for Review** then the claim can be only loaded and shown to the user without any subsequent action.
+
+
+      The page will open with the current information loaded into the data entry fields. See the `Claim Review Page <#claim-review-page>`__, for information on the data entry and mandatory fields.
```
Comment (kmadej) [inline _include/um/claims/review_claims.rst:212]: the page will be opened or 'the page will open the current information...'
'

File: _include/um/claims/review_claims.rst [hunk 8]
```diff
@@ -0,0 +1,594 @@
...
+
+    If inappropriate data is entered at the time the user clicks the ``Save`` or `` review`` button, an error message will appear in the Information Panel, and the data field will take the focus.
+
+ #. **Cancel**
+
+    By clicking on the ``Cancel`` button, the user will be re-directed to the `Claims Overview Page <#claims-overview-page>`__.
+
+Claim Feedback Page
+"""""""""""""""""""
+    will show read-only information of the current claim selected for feedback, on the top section of the page it has input boxes where a user with the role Medical Officer can enter feedback on the current claim or where the user can read a feedback delivered by enrolment officers.
```
Comment (kmadej) [inline _include/um/claims/review_claims.rst]: It will show

File: _include/um/claims/review_claims.rst [hunk 9]
```diff
@@ -0,0 +1,594 @@
...
+
+   Read-only data of the feedback includes in the section **Claim** the following:
+
+    * ``HF Code``
+
+      The health facility code which the claim belongs to.
+
+    * ``HF Name``
+
+      The health facility name which the claim belongs to
```
Comment (kmadej) [inline _include/um/claims/review_claims.rst]: add dot to the end of sentence

File: _include/um/insuree_policies/family.rst [hunk 10]
```diff
@@ -0,0 +1,511 @@
...
+  .. _image124:
+  .. figure:: /img/user_manual/image98.png
+    :align: center
+
+    `Image 124 - Change Family/Group Page`
+
+Changing a Head of Family/Group
+"""""""""""""""""""""""""""""""
+
+  The head of the Family/Group is the main contact associated with a policy. For various reasons it may be necessary to change the head of a family/group. The new head must a head of family in another family.
```
Comment (kmadej) [inline _include/um/insuree_policies/family.rst]: something wrong with this sentence - 'The new head must a head of family in another family.'

must be a head ?

File: _include/um/register/claim_admin.rst [hunk 11]
```diff
@@ -0,0 +1,232 @@
...
+  .. _image68:
+  .. figure:: /img/user_manual/image56.png
+    :align: center
+
+    `Image 68 - Claim Administrators Control Page`
+
+Claim Administrators Control Page
+"""""""""""""""""""""""""""""""""
+
+  The ``Claim Administrators Control Page`` is the central point for all claim administrators administration. By having access to this panel, it is possible to add, edit, delete and search claim administrators. The panel is divided into four panels (:ref:`Image 68<image68>`).
```
Comment (kmadej) [inline _include/um/register/claim_admin.rst]: something wrong here 'The ``Claim Administrators Control Page`` is the central point for all claim administrators administration'

administartors adminstration should be improved i suppose

File: _include/um/register/enrolment_officer.rst [hunk 12]
```diff
@@ -0,0 +1,290 @@
...
+
+      Enter the e-mail address for the Village Executive officer. 50 characters maximum.
+
+    * ``Date of Birth``
+
+      Enter the date of birth for the Village Executive officer. *Note. To clear the date entry box; use the mouse to highlight the full date and then press the space key.*
+
+    * ``Include Login``
+
+      Enable the enrolement officer to use openIMIS web by creating web credentials (enrolement officers primary interface is the enrolment application running on android device that doesn't require openIMIS user), the change must be saved to be taken into account.
```
Comment (kmadej) [inline _include/um/register/enrolment_officer.rst]: enrolement -> enrolment

File: _include/um/register/item_price_list.rst [hunk 13]
```diff
@@ -0,0 +1,197 @@
+Medical Item Price Lists Administration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+  Pricelists of medical items are tools for specification which medical items and at which prices can be invoiced by contractual health facilities to the scheme administration. Administration of pricelists of medical items is restricted to users with the role of Scheme Administrator.
+
+Pre-conditions
+""""""""""""""
+
+  A price list of medical items may only be added, after an agreement with a health facility or a group of health facilities on specific prices. Editing of the price list may occur only after an approval of the management of the scheme administration. Deletion of a price list of medical Items normally will occur when a price list becomes obsolete.
```
Comment (kmadej) [inline _include/um/register/item_price_list.rst]: in first sentence 'medical items' is lowercase however in last sentence Items is uppercase

File: _include/um/register/policy.rst [hunk 14]
```diff
@@ -0,0 +1,662 @@
...
+
+      Displays the code for the medical service
+
+    * ``Name``
+
+      Displays the name of the medical service
+
+    * ``Type``
+
+      Displays the type of the medical service\
```
Comment (kmadej) [inline _include/um/register/policy.rst]: remove '\' at the end

**Review by kmadej — CHANGES_REQUESTED** [review]

File: _include/um/claims/review_claims.rst (hunk 7 above)
Comment (dragos-dobre) [reply _include/um/claims/review_claims.rst:212]: I think this is correct as the page opens and the current information will be loaded into the fields.

**Review by kmadej — APPROVED** [review]

**Pull request merged**